ext/soap/tests/bug69280.wsdl

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<definitions xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:s="http://www.w3.org/2001/XMLSchema" xmlns:http="http://schemas.xmlsoap.org/wsdl/http/" xmlns:mime="http://schemas.xmlsoap.org/wsdl/mime/" xmlns:tm="http://microsoft.com/wsdl/mime/textMatching/" xmlns:s0="http://tempuri.org/" name="TestWS" targetNamespace="http://tempuri.org/" xmlns="http://schemas.xmlsoap.org/wsdl/">
+	<types>
+		<xs:schema elementFormDefault="qualified" targetNamespace="http://tempuri.org/" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+			<xs:complexType name="AbstractClass">
+				<xs:sequence>
+					<xs:element minOccurs="0" maxOccurs="1" name="prop" type="xs:string" />
+				</xs:sequence>
+			</xs:complexType>
+			<xs:complexType name="RealClass1">
+				<xs:complexContent mixed="false">
+					<xs:extension base="s0:AbstractClass">
+						<xs:sequence>
+							<xs:element minOccurs="0" maxOccurs="1" name="prop1" type="xs:string" />
+						</xs:sequence>
+					</xs:extension>
+				</xs:complexContent>
+			</xs:complexType>
+		</xs:schema>
+	</types>
+	<message name="TestMethodSoapIn">
+		<part name="parameters" element="RealClass1" />
+	</message>
+	<portType name="TestWSSoap">
+		<operation name="TestMethod">
+			<input message="s0:TestMethodSoapIn" />
+		</operation>
+	</portType>
+	<binding name="TestWSSoap" type="s0:TestWSSoap">
+		<soap:binding transport="http://schemas.xmlsoap.org/soap/http" />
+		<operation name="TestMethod">
+			<soap:operation soapAction="http://tempuri.org/TestMethod" style="document" />
+			<input>
+				<soap:body use="literal" />
+			</input>
+			<output>
+				<soap:body use="literal" />
+			</output>
+		</operation>
+	</binding>
+	<service name="TestWS">
+		<port name="TestWSSoap" binding="s0:TestWSSoap">
+			<soap:address location="http://tempuri.org/" />
+		</port>
+	</service>
+</definitions>
```

Comment:
Do we use 4 spaces for XML indent?

Reply:
The test files are inconsistent, some use 2 spaces, some 4 spaces, and some tabs... In the end it doesn't really matter though.